src/Altinn.Platform/Altinn.Platform.Storage/DataCleanup/NightlyCleanup.cs

```diff
@@ -1,23 +1,67 @@
 using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
 
+using Altinn.Platform.Storage.DataCleanup;
+using Altinn.Platform.Storage.DataCleanup.Services;
+using Altinn.Platform.Storage.Interface.Models;
 using Microsoft.Azure.WebJobs;
+using Microsoft.Azure.WebJobs.Hosting;
 using Microsoft.Extensions.Logging;
 
+[assembly: WebJobsStartup(typeof(Startup))]
```

Comment:
The assemblyattribute could be moved to the Startup.cs file.

Reply:
done